sparse/tests/test_io.py

```diff
@@ -11,8 +11,9 @@
 
 
 @pytest.mark.parametrize("compression", [True, False])
-def test_save_load_npz_file(compression):
-    x = sparse.random((2, 3, 4, 5), density=0.25)
+@pytest.mark.parametrize("format", ["coo", "gcxs"])
+def test_save_load_npz_file(compression, format):
```

Comment:
Can you rewrite this using `tmp_path`: https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture

Reply:
Same for the next test.